amlip_cpp/include/amlip_cpp/node/collaborative_learning/ModelManagerReceiverNode.hpp

```diff
@@ -159,6 +159,15 @@ class ModelManagerReceiverNode : public ParentNode
      */
     AMLIP_CPP_DllAPI void stop();
 
+    /**
+     * @brief Request the model that is associated with the given statistics.
```

Comment:
We are actually requesting the latest model available. I mean that we are not sending back the statistics so we can request the model for those specific statistics. I do not say that this is wrong, but we should know it. For now, maybe just adding a comment about this should be fine.